activerecord/test/cases/adapter_test.rb

```diff
@@ -442,6 +442,20 @@ def setup
       end
     end
 
+    def test_create_with_query_cache
+      @connection.enable_query_cache!
+
+      count = Post.count
+      binds = ["", ""]
+      bind_param = Arel::Nodes::BindParam.new(nil)
```

Comment:
Can you remove unused variables?

```suggestion
```

https://github.com/rails/rails/pull/41147/checks?check_run_id=1716707664

Reply:
Done